jstest/testrunner/devices/serial_device.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2018-present Samsung Electronics Co., Ltd. and other contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from jstest.common import console
+from jstest.testrunner import utils as testrunner_utils
+from jstest.testrunner.devices.device_base import RemoteDevice
+from jstest.testrunner.devices.connections.serialcom import SerialConnection
+
+
+class SerialDevice(RemoteDevice):
```

Comment:
You can add a minimal description about the class. E.g. Common superclass for serial devices.